maisim/maisim.Game/Component/Gameplay/Notes/MaisimNotes.cs

```diff
@@ -0,0 +1,12 @@
+﻿using osu.Framework.Graphics.Containers;
+
+namespace maisim.Game.Component.Gameplay.Notes
+{
+    /// <summary>
+    /// A base abstract class for notes in the gameplay.
+    /// </summary>
+    public abstract class MaisimNotes : CompositeDrawable
```

Comment:
Rename to `DrawableNote`, current name doesn't imply that the class is a displayable element